examples/CreateSimpleEncoding.php

```diff
@@ -31,7 +31,7 @@
 $videoStreamConfig_1080 = new H264VideoStreamConfig();
 $videoStreamConfig_1080->input = new HttpInput($videoInputPath);
 $videoStreamConfig_1080->width = 1920;
-$videoStreamConfig_1080->height = 1080;
+$videoStreamConfig_1080->height = 816;
```

Comment:
because the default input file comes with aspect ratio 40:17 instead of 16:9. Using 1080 stretches the resulting video content